CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # go-ipfs changelog
 
+Ipfs 0.4.16 is a fairly small release in terms of changes to the ipfs codebase,
+but it contains a huge amount of changes and improvements from libraries we
+depend on, notably libp2p. This version of ipfs contains the changes for libp2p
+v6.0.5, In that release, we made significant changes to the codebase to allow
+for more easy integration of future transports and modules. We also improved
+some of our dialing logic, fixed a couple issues in the DHT, and added
+support for a new way to write ipfs multiaddrs.  The transport refactor now
+allows us to much more easily add in support for running libp2p (and by
+extension, ipfs) over QUIC, or using TLS instead of secio for encrypting
+connections. Our [QUIC
+transport](https://github.com/libp2p/go-libp2p-quic-transport) currently
+works, and can be plugged into libp2p manually (though note that it is
```

Comment:
Is there a doc, issue, or discuss post, or blog post we can point people to that describes how to add the QUIC transport manually?

Reply:
No. Really, we don't want users to do this unless the know what they're doing (or ask us and we can talk it over with them).